.github/workflows/check_compatibility.yml

```
@@ -0,0 +1,44 @@
+name: Check compatibility
+
+on:
+  pull_request:
+
+jobs:
+  check-interface-compatibility:
```

nit since it shows up in the UI

```suggestion
  check-compatibility:
```